docs/projects/index.md

```diff
@@ -60,6 +60,10 @@ The `contractingProcesses` array can be used to provide a summary of the procure
 
 The `contractingProcesses/modifications` section can be used to record information on changes to each contracting process.
 
+## When to publish OC4IDS?
+
+ A project does not need to be complete before publishing in OC4IDS. The data can be updated at each stage of a project or when there is a change.   
+ 
```

Comment:
I'm not sure this is quite the right place for the guidance since the subheadings immediately before and after are more about the structure of OC4IDS data.

Also, it's better to lead with what the reader should do (publish updates throughout the project lifecycle) rather than what they shouldn't do (wait until the project is complete before publishing updates).